src/main/java/org/broadinstitute/barclay/argparser/TaggedArgumentParser.java

```diff
@@ -142,11 +143,22 @@ private void replaceTaggedOption(
             } else {
                 // the option appears to be tagged, but we're already at the end of the argument list,
                 // and there is no companion value to use
-                throw new CommandLineException("No value found for tagged argument: " + optionString);
+                throw new CommandLineException("No argument value found for tagged argument: " + optionString);
             }
         }
     }
 
+    // Reject attempts to use hybrid Barclay/legacy syntax that contains embedded "=". Most of the time
+    // this works because jopt accepts "-O=value". But if "value" contains what appears to be tagging
```

Comment:
Why not just use a javadoc comment?  I've never really understood the prohibition on putting javadoc on private things.

Reply:
Just due to habit. Fixed.